cfg/conf.sample.php

```diff
@@ -258,4 +263,5 @@ class = Filesystem
 ; (optional) the "signature" (access key) issued by YOURLS for the using account
 ; signature = ""
 ; (optional) the URL of the YOURLS API, called to shorten a PrivateBin URL
-; apiurl = "https://yourls.example.com/yourls-api.php"
\ No newline at end of file
+; apiurl = "https://yourls.example.com/yourls-api.php"
+*/
```

Comment:
It is not necessary to close the PHP-comment and arguably leads to the risk that someone adds new settings below that end-of-comment block and gets a surprise when those end up getting leaked. It is safer not to add it, even if syntactically not correct. And it would be even safer not to expose the cfg directory, as we recommend in the installations hardening section - the PHP-comment is a belt-and-braces mechanism to further protect the content of this file.

```suggestion
```

Reply:
Done